scripts/build_windows.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+function cleanup {
+  shred --force --remove --zero --iterations=10 CLIEVCodeSigningCertificate2.pfx
```

Comment:
Probably not the end of the world if we don't do this since the CI machines get reset between runs.

Reply:
True, but I'm writing this so far on the assumption that we might be releasing before we fully switch to the CI.